app/views/glossary_terms/show.html.erb

```diff
@@ -51,4 +51,10 @@
     <%= :footer_created_at.t(date: @glossary_term.created_at.web_time) %><br/>
     <%= :footer_last_updated_at.t(date: @glossary_term.updated_at.web_time) %>
   </p>
+
+  <div id="glossary_term_authors_editors">
+    <%= panel_block do %>
+      <%= show_authors_and_editors(obj: @glossary_term, user: @user) %>
+    <% end %>
+  </div>
```

Comment:
I find it always time-consuming to write the view.
But this one was relatively quick, thanks to @nimmolo's helper methods and the pattern he used for Name and Location.

Reply:
Glad it helped. 

(Of course, I probably changed it for the bootstrap 5 branch, but it should be easy enough to adapt this and the other Glossary Term PR, and keep that branch up with these PRs.)